server/tests/audit_math/test_bravo.py

```diff
@@ -563,14 +639,19 @@ def test_tied_contest():
         "0.8": {"type": None, "size": 41, "prob": 0.8},
         "0.9": {"type": None, "size": 57, "prob": 0.9},
     },
-    "test3": {"asn": {"type": "ASN", "size": 1, "prob": 1.0},},
-    "test4": {"asn": {"type": "ASN", "size": 1, "prob": 1.0},},
-    "test5": {
-        "asn": {"type": "ASN", "size": 1000, "prob": 1},
-        "0.7": {"type": None, "size": 1000, "prob": 0.7},
-        "0.8": {"type": None, "size": 1000, "prob": 0.8},
-        "0.9": {"type": None, "size": 1000, "prob": 0.9},
+    "test3": {
+        "asn": {"type": "ASN", "size": -1.0, "prob": -1.0},
```

Comment:
why did these all change to size -1.0? seems weird that it's negative and that it's a float

Reply:
you're right about the float. The -1 is because these are single-candidate races and invalid. I could throw an exception instead.

Reply:
Exceptions seem clearer to me